lektor/imagetools.py

```diff
@@ -3,9 +3,17 @@
 import imghdr
 import struct
 import posixpath
+import warnings
 from datetime import datetime
 from xml.etree import ElementTree as etree
+import decimal
 import exifread
+try:
+    from enum import IntEnum
```

Comment:
When does this fail? I can import this in 2.7 and 3.6. The failure isn't covered by the tests either.

Reply:
Ah, you're right, I did have it. I want to include enum34 so we're actually using enums on 2.7.

My understanding of the codecov report must be off. I though it took into account all testing environments, so it would have hit the exception on the 2.7 tests if enum wasn't available, which it shouldn't have been. So I got confused.